docs/website/content/v0.6/en/guides/getting-started/talosctl.md

```diff
@@ -9,10 +9,8 @@ This guide gives you some hands on examples, and some more context when working
 
 To get going with `talosctl` you need to download the latest release from Github [here](https://github.com/talos-systems/talos/releases).
 
-> This might be an older release when you're reading this guide, always check for the latest version.
-
 ```bash
-wget https://github.com/talos-systems/talos/releases/download/v0.4.1/talosctl-linux-amd64
+https://github.com/talos-systems/talos/releases/download/latest/talosctl-darwin-amd64
```

Comment:
Looks like `wget` was dropped, and the linux was replaced with darwin. Will push a fix in a bit.